controllers/territory_controller/tests.py

```diff
@@ -204,16 +204,21 @@ def load_territory_owners(self, claim_log: ClaimLog) -> None:
 
     @patch('controller.Supervisor.getFromDef')
     @patch('territory_controller.get_robot_device')
-    def setUp(self, mock_get_robot: Mock, mock_getFromDef: Mock) -> None:
+    @patch('territory_controller.TerritoryController.set_score_display')
+    def setUp(
+        self,
+        mock_score_display: Mock,
+        mock_get_robot: Mock,
+        mock_getFromDef: Mock,
```

Comment:
Since we're not using these I'd probably call them `_`, `__` and `___` (they'll need to be unique I think) and give then all an annotation of `object`. (also applies to the ones on the tests).

Reply:
21740203e9f877176115e141713ac0e64a38fd5c